toolchain/mfc/gen_case_constraints_docs.py

```
@@ -710,25 +714,71 @@ def _render_cond_parts(trigger_str, cond_dict):
+            "They catch common mistakes in initial conditions and EOS parameters.\n"
+        )
+
+        # Group by method
+        warnings_by_method: Dict[str, List[Rule]] = defaultdict(list)
+        for r in all_warnings:
+            warnings_by_method[r.method].append(r)
+
+        # Method name -> human-readable title
+        method_titles = {
+            "check_volume_fraction_sum": "Volume Fraction Sum",
+            "check_alpha_rho_consistency": "Alpha–Rho Consistency",
```

Ruff (0.15.0)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)